.github/workflows/test-internal-prs.yml

```diff
@@ -5,15 +5,15 @@
 #   (2) On every commit to the PR
 #   (3) Adding run-tests label to the PR
 
-name: Test / Internal PRs
```

Comment:
Can we rename the file to `test-next-release-prs.yml`? GHA often has trouble deduping workflows if the filenames are the same.

Reply:
oh okay